app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -295,6 +316,7 @@ export default class AMRHeatmapView extends React.Component {
         </p>
       );
     }
+    logAnalyticsEvent("AMRHeatmapView_heatmap_rendered");
```

Comment:
Same as comment above, with logging a render. I think we can infer the page views by searching for the matching http get requests for `/amr_heatmap`.